dashboard/test/dsl/dsl_test.rb

```diff
@@ -31,6 +31,17 @@ class DslTest < ActiveSupport::TestCase
     assert_equal i18n_expected, i18n
   end
 
+  test 'test Script DSL admin_required ' do
+    input_dsl = "
+admin_required true
```

Comment:
ha, it's supposed to be a string, it coincidentally works for boolean true also. I will add another test so we have a test with both.